xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -828,8 +828,15 @@ void CRenderManager::FlipPage(volatile std::atomic_bool& bStop, double pts /* =
         presentmethod = PRESENT_METHOD_BOB;
         invert = true;
       }
+      else if (interlacemethod == VS_INTERLACEMETHOD_IMX_ADVMOTION_DOUBLE)
```

Comment:
@FernetMenta 

reviewed the whole process again - simplified the logic, functionality stays, even removed the last platform specific method from RenderManager's code. 

can you please do a quick check now? the relevant squash is https://github.com/xbmc/xbmc/pull/9253/commits/f450b9e6ae6bc2d0752a2083cdab6379e7c9a4fa